artifacts/definitions/SUSE/Linux/Events/SystemLogins.yml

```diff
@@ -0,0 +1,140 @@
+name: SUSE.Linux.Events.SystemLogins
+description: |
+   Pulls all the system login and logout events from auditd logs.
+   
+type: CLIENT_EVENT
+
+sources:
+  - name: SuccessfulLogins
+    precondition:
+      SELECT OS From info() where OS = 'linux'
+
+    query: |
+      LET SYSTEM_LOGINS = SELECT Timestamp, Sequence, Result, Session, str(str=Category) as auth_category, str(str=Type) as auth_type, Data.acct as Account, Data.op as Method, Data.addr as Source_IP, Data.hostname as Source_Hostname, Data.terminal as Terminal FROM audit() WHERE auth_category = 'user-login' AND auth_type = 'USER_AUTH' AND Result = 'success'
+      SELECT * FROM foreach(row=SYSTEM_LOGINS, query={SELECT Timestamp, Sequence, Result, Session, Method, Account as User, Source_IP, Source_Hostname, Terminal FROM scope()})
+
+  - name: FailedLogins
+    precondition:
+      SELECT OS From info() where OS = 'linux'
+
+    query: |
+      LET SYSTEM_LOGINS = SELECT Timestamp, Sequence, Result, Session, str(str=Category) as auth_category, str(str=Type) as auth_type, Data.acct as Account, Data.op as Method, Data.addr as Source_IP, Data.hostname as Source_Hostname, Data.terminal as Terminal FROM audit() WHERE auth_category = 'user-login' AND auth_type = 'USER_AUTH' AND Result = 'fail'
+      SELECT * FROM foreach(row=SYSTEM_LOGINS, query={SELECT Timestamp, Sequence, Result, Session, Method, Account as User, Source_IP, Source_Hostname, Terminal FROM scope()})
+
+  - name: SuccessfulLogouts
+    precondition:
+      SELECT OS From info() where OS = 'linux'
+
+    query: |
+      LET SYSTEM_LOGINS = SELECT Timestamp, Sequence, Result, Session, str(str=Category) as auth_category, str(str=Type) as auth_type, Data.acct as Account, Data.op as Method, Data.addr as Source_IP, Data.hostname as Source_Hostname, Data.terminal as Terminal FROM audit() WHERE auth_category = 'user-login' AND auth_type = 'USER_END' AND Result = 'success'
+      SELECT * FROM foreach(row=SYSTEM_LOGINS, query={SELECT Timestamp, Sequence, Result, Session, Method, Account as User, Source_IP, Source_Hostname, Terminal FROM scope()})
+
+reports:
+  - type: MONITORING_DAILY
```

Comment:
Does this mean that it only runs the report daily?

Reply:
According to https://github.com/SUSE/linux-security-sensor/blob/main/reporting/doc.go#L16-L20, it does seem like it's running it daily, though I am not sure if the graphs get cleared after 24 hours since I haven't not kept my vagrant dev instance running that long. I basically used the same mechanism in the SSHLogin artifact since they seem similar. https://github.com/SUSE/linux-security-sensor/blob/main/artifacts/definitions/Linux/Events/SSHLogin.yaml#L53